src/Controller/SentencesController.php

```diff
@@ -828,6 +833,61 @@ public function edit_correctness()
         }
     }
 
+    /**
+     * Mark all sentences of a user as incorrect.
+     *
+     * @param string $userId ID of the user. 
+     * 
+     * @return void
+     */
+    public function mark_unreliable($userId)
+    {   
+        $this->loadModel('Users');
+        $user = $this->Users->getUserById($userId);
+
+        if(empty($user)) {
+            $this->set("userExists", false);
+            $this->redirect(
+                array(
+                    "controller" => "sentences",
+                    "action" => "of_user",
+                    $user['username']
+                )
+            );
+            return;
+        }
+
+        if (CurrentUser::canMarkSentencesOfUser($user)) {
```

Comment:
Sorry that I wasn't clear enough but you don't need all these lines since only admins are allowed to access this controller action (in `config/auth_actions.php`).

Reply:
Ah, got you now.